versions/3.0.md

```diff
@@ -849,7 +849,7 @@ A path parameter of a string value:
 {
   "name": "username",
   "in": "path",
-  "description": "username to fetch",
+  "description": "user name to fetch",
```

Comment:
I'm no native speaker of English, but I thought English would use much less composite words than my mother tongue (German).
Is this an actual English word here, or just repeating the variable name?